src/app/pages/reports-dashboard/components/exporters/reporting-expoters-list/reporting-exporters-list.component.ts

```diff
@@ -92,12 +107,17 @@ export class ReportingExporterListComponent implements OnInit {
     private dialogService: DialogService,
     protected emptyService: EmptyService,
     private appLoader: AppLoaderService,
+    private layoutService: LayoutService,
   ) {}
 
   ngOnInit(): void {
     this.getExporters();
   }
 
+  ngAfterViewInit(): void {
```

Comment:
There is now a `ixPageHeader` directive to replace this.

Reply:
@undsoft - I'll manage it in a separate ticket checking other instances where we have the same logic.